A36-xds-for-servers.md

```diff
@@ -148,22 +193,28 @@ result that can be hard-coded. This applies to `destination_port` which relies
 on `use_original_dst`. It also applies to `server_names`, `transport_protocol`,
 `application_protocols` which depend on `Listener.listener_filters`.
 
+### Language-specifics
+
+The overall "wrapping" server API has many language-specific ramifications. We
+show each individual language's approach. For C++ and wrapped languages we just
+show a rough sketch of how they would be done and expect a followup gRFC for
```

Comment:
We'll actually probably just backfill the details into this gRFC, rather than creating a separate one.